### ISSUE TITLE
Unable to resolve threads in merge request [Gitlab Self-Managed] [JetBrains plugin]

### ISSUE THREAD

**Issue by lokinz** [TeamCodeStream/codestream#1043]
- IDE Version: PhpStorm 2022.1.2
- OS Version: openSUSE Tumbleweed
- Plugin Version: 13.1.1+237
- Integration: GitLab Self-Managed 

Steps to Reproduce:

1. Select PR Details from the Merge Requests section
2. Find an open thread
3. Try resolving thread by clicking the check-mark icon or the "Resolve thread" button. 
4. Notice the update does not happen

Logs: 
- idea.log
```
WARN - #c.c.a.CodeStreamLanguageClient - [2022-07-01 04:03:57:662] executeMethod failed — {"method":"resolveReviewThread"}
Error: Type mismatch on variable $id and argument id (ID! / DiscussionID!): {"response":{"errors":[{"message":"Type mismatch on variable $id and argument id (ID! / DiscussionID!)","locations":[{"line":3,"column":35}],"path":["mutation DiscussionToggleResolve","discussionToggleResolve","input","id"],"extensions":{"code":"variableMismatch","variableName":"id","typeName":"ID!","argumentName":"id","errorMessage":"Type mismatch"}}],"status":200,"headers":{}},"request":{"query":"\n\t\tmutation DiscussionToggleResolve($id:ID!, $resolve: Boolean!) {\n\t\t\tdiscussionToggleResolve(input:{id:$id, resolve:$resolve}) {\n\t\t\t\t  clientMutationId\n\t\t\t\t\t  discussion {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tresolvedAt\n\t\t\t\t\t\tresolved\n\t\t\t\t\t\tresolvable\n\t\t\t\t\t\tresolvedBy {\n\t\t\t\t\t\t\t  login: username\n\t\t\t\t\t\t\t  avatarUrl\n\t\t\t\t\t\t}\n\t\t\t\t\t\tnotes {\n\t\t\t\t\t\t\tnodes {\n\t\t\t\t\t\t\t  id\n\t\t\t\t\t\t\t  resolvable\n\t\t\t\t\t\t\t  resolved\n\t\t\t\t\t\t\t}\n\t\t\t\t\t\t}\n\t\t\t\t\t  }\n\t\t\t\t  }\n\t\t\t  }","variables":{"id":"gid://gitlab/DiffDiscussion/d59ceda3dd5b7190d0b625903a75e240f95b2dfe","resolve":true}}}
Error: Type mismatch on variable $id and argument id (ID! / DiscussionID!): {"response":{"errors":[{"message":"Type mismatch on variable $id and argument id (ID! / DiscussionID!)","locations":[{"line":3,"column":35}],"path":["mutation DiscussionToggleResolve","discussionToggleResolve","input","id"],"extensions":{"code":"variableMismatch","variableName":"id","typeName":"ID!","argumentName":"id","errorMessage":"Type mismatch"}}],"status":200,"headers":{}},"request":{"query":"\n\t\tmutation DiscussionToggleResolve($id:ID!, $resolve: Boolean!) {\n\t\t\tdiscussionToggleResolve(input:{id:$id, resolve:$resolve}) {\n\t\t\t\t  clientMutationId\n\t\t\t\t\t  discussion {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tresolvedAt\n\t\t\t\t\t\tresolved\n\t\t\t\t\t\tresolvable\n\t\t\t\t\t\tresolvedBy {\n\t\t\t\t\t\t\t  login: username\n\t\t\t\t\t\t\t  avatarUrl\n\t\t\t\t\t\t}\n\t\t\t\t\t\tnotes {\n\t\t\t\t\t\t\tnodes {\n\t\t\t\t\t\t\t  id\n\t\t\t\t\t\t\t  resolvable\n\t\t\t\t\t\t\t  resolved\n\t\t\t\t\t\t\t}\n\t\t\t\t\t\t}\n\t\t\t\t\t  }\n\t\t\t\t  }\n\t\t\t  }","variables":{"id":"gid://gitlab/DiffDiscussion/d59ceda3dd5b7190d0b625903a75e240f95b2dfe","resolve":true}}}
    at /home/me/.codestream/agent/agent-13.1.1+237.js:2:897075
    at /home/me/.codestream/agent/agent-13.1.1+237.js:2:891954
    at Object.next (/home/me/.codestream/agent/agent-13.1.1+237.js:2:892059)
    at o (/home/me/.codestream/agent/agent-13.1.1+237.js:2:890773)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    
```

I have tried this with threads created from both GitLab web and from within the IDE. 
Please let me know if I can help with any more information. 

Thanks.

**Comment by planteater** [comment]
Thanks for the report @lokinz! Could I trouble you for the complete log file? You can send it to me directly at support@codestream.com.

**Comment by lokinz** [comment]
Thanks. I have emailed the complete log.

**Comment by planteater** [comment]
@lokinz - This should be fixed in v13.3 of the CodeStream extension.
